conf/rubocop.yml

```diff
@@ -20,5 +15,21 @@ RSpec/DescribeMethod:
 RSpec/ExampleLength:
   Enabled: false
 
+RSpec/ExpectActual:
+  Exclude:
+  - 'spec/routing/**/*'
```

Comment:
Routing specs tend to use literals as the argument to `expect`:

https://github.com/backus/rubocop-rspec/issues/198

Not that we tend to write a lot of routing specs. I can add a link to that issue here in the config.